core/Cookie.php

```diff
@@ -132,7 +134,7 @@ protected function getDefaultExpire()
      * @param bool $Secure
      * @param bool $HTTPOnly
      */
-    protected function setCookie($Name, $Value, $Expires, $Path = '', $Domain = '', $Secure = false, $HTTPOnly = false)
+    protected function setCookie($Name, $Value, $Expires, $Path = '', $Domain = '', $Secure = false, $HTTPOnly = false, $sameSite = null)
```

Comment:
not too important but usually the other default values that aren't set are `false`... (null be generally better it's just for consistency be good to use `false`)